x-pack/platform/plugins/private/product_intercept/kibana.jsonc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "plugin",
+  "id": "@kbn/product-intercept-plugin",
+  "owner": ["@elastic/appex-sharedux"],
+  "group": "platform",
+  "visibility": "private",
```

Comment:
Same comment as for the intercepts plugin.

Reply:
It's fine to have the product intercept visibility as private, other plugins shouldn't depend on this plugin.